tools/Customization/DevHome.FileExplorerSourceControlIntegration/DevHome.FileExplorerSourceControlIntegration.csproj

```diff
@@ -13,6 +12,7 @@
         <CsWinRTEnabled>false</CsWinRTEnabled>
         <Platforms>x86;x64;arm64</Platforms>
         <RuntimeIdentifiers>win-x86;win-x64;win-arm64</RuntimeIdentifiers>
+		<Nullable>enable</Nullable>

```

Comment:
The format looks odd here but I can see it is as expected locally.

Reply:
Replace tabs with spaces.